src/17-infinite-discovery/data.md

```diff
@@ -1,5 +1,48 @@
 # How to generate the data
 
+## Quick way to generate the data
+
+## Load the production console
+
+```bash
+  bundle exec rails console production
+```
+
+```ruby
+  # assuming you want to generate the data for the artworks in the curators picks collection
+  collection = MarketingCollection.find('curators-picks')
+  artworks = collection.artworks.published.for_sale
+  artworks_json = JSON.pretty_generate(artworks.map do |w|
+    {
+      id: w.id,
+      slug: w.slug,
+      title: w.title,
+      date: w.date,
+      rarity: w.attribution_class,
+      medium: w.category,
+      materials: w.medium,
+      list_price_amount: w.price_listed,
+      list_price_currency: w.price_currency,
+      categories: w.total_genome.without("Art", "Career Stage Gene").select{ |k,v| k !~ /(galleries based|made in)/i && v == 100}.keys,
+      tags: w.tags + w.auto_tags,
+      additional_information: w.additional_information,
+      image_url: w.default_image.image_urls['large'],
+      colors: w.colors,
+      artist_name: w.artists.first&.name,
+      artist_nationality: w.artists.first&.nationality,
+      artist_birthday: w.artists.first&.birthday,
+      artist_gender: w.artists.first&.gender,
+    }
+  end)
+  s3_client = S3.new(bucket: "artsy-public")
+  s3_client.store_object("curater_picks.json", json, "text/json", "public-read")
+```
+
+Download the json file from the
+[public s3 bucket](https://us-east-1.console.aws.amazon.com/s3/object/artsy-public?region=us-east-1&bucketType=general&prefix=curater_picks.json).
```

Comment:
Do we want it to be in a non-public s3 bucket? Whats public now might not be public in a week or month, right? Just want to make sure we don't forget about this and have it hanging around accidentally?

Reply:
Good point. I’m honestly not sure what the best approach is here. On one hand, this data is already publicly available on artwork pages, so I wasn’t too concerned about privacy

Reply:
developers will have access to private s3 buckets, so just add to one of those?